tracpro/baseline/forms.py

```diff
@@ -0,0 +1,55 @@
+from django import forms
+
+from tracpro.polls.models import Poll
+from .models import BaselineTerm
+
+
+class BaselineTermForm(forms.ModelForm):
+    """
+    Form for Baseline Term
+    """
```

Comment:
Would it be difficult to use javascript to hide the `Baseline Question` and `Follow up Question` fields until the corresponding `Poll` field has data?